Gemfile

```diff
@@ -39,7 +39,7 @@ gem 'hydra-role-management', '~> 0.2.0'
 gem 'rsolr', '~> 1.1.0'
 gem 'devise', '~> 3.0'
 gem 'devise-guests', '~> 0.3'
-gem 'iiif-presentation'
+gem 'iiif-presentation', github: 'tpendragon/osullivan', branch: 'no_singleton_classes'
```

Comment:
I'd like to get it in today at least, because CI fails a bunch without some of this.